roles/edge-image-builder/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Create eib images directory
+  become: true
+  file:
+    path: "{{ eib_path }}/images"
+    state: directory
+
+- name: Copy OS image to eib images directory
+  become: true
```

Comment:
Is become really needed?

Reply:
same answer...I will test it again without that part

Reply:
tested